oss-internship-2020/pffft/test_pffft_sandboxed.cc

```diff
@@ -0,0 +1,180 @@
+#include <assert.h>
+#include <math.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/times.h>
+#include <syscall.h>
+#include <time.h>
+
+#include "fftpack.h"
+#include "pffft_sapi.sapi.h"
+#include "sandboxed_api/util/flag.h"
+#include "sandboxed_api/vars.h"
+
+ABSL_DECLARE_FLAG(string, sandbox2_danger_danger_permit_all);
+ABSL_DECLARE_FLAG(string, sandbox2_danger_danger_permit_all_and_log);
+
+class pffftSapiSandbox : public pffftSandbox {
```

Comment:
I think it'd be better to rename those both files 'test_pffft.cc' and 'test_pffft_sandboxed.cc' to maybe just 'pffft.cc' and 'sandboxed_pffft.cc'? Otherwise readers of this code might think these are some unittest files?